### ISSUE TITLE
Missing alt-text for button (WCAG 1.1.1a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1217]
**Request:** The intent of this Success Criterion is to make information conveyed by non-text content accessible through the use of a text alternative. Text alternatives are a primary way for making information accessible because they can be rendered through any sensory modality (for example, visual, auditory or tactile) to match the needs of the user.

**Issue:**
1. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/437649-3/?language=en-/slide-719796-3
- The menu button doesn´t have an alternative text.
--> An alt-text should be provided by `aria-label` or `title`.

**Comment by rezatavakoli** [comment]
This button has been removed in the latest version. Therefore, this issue has been closed.
